ui/app/styles/components/secure-variables.scss

```diff
@@ -27,6 +27,8 @@
   }
 
   .path-input {
+    height: 2.25em;
```

Comment:
`app/styles/core/forms.scss` sets the default height of `input` to be `auto` which overrides the normal `2.25 em` expected in the `Input` component which is also how `SingleSelectDropdown` has its height set to.